spec/02-integration/05-proxy/09-balancer_spec.lua

```diff
@@ -305,6 +305,7 @@ end
 local localhosts = {
   ipv4 = "127.0.0.1",
   ipv6 = "0000:0000:0000:0000:0000:0000:0000:0001",
+  localhost = "localhost",
```

Comment:
Because we refer to the key as `ipv`, shouldn't this key be `hostname`? This way we highlight that we are testing IPv4, IPv6, and hostnames. I was originally confused reading this and the below loop.

Reply:
Changed the key to `hostname` and the other variable to `mode` (a "meh" name but now more correct than `ipv`) to reduce the confusion!